### ISSUE TITLE
Feature request: Deamon Support

### ISSUE THREAD

**Issue by alberto467** [cardigann/cardigann#244]
### Description of feature/enhancement
I'd really like a command line option to make cardigann run without popping up the command prompt.

### Justification
It's annoying to have a command line window for a task that could be executed in the background.

### Example use case
You open the executable with the `-d` or `--deamon` option and it runs in the background.

**Comment by lxt2** [comment]
See https://github.com/cardigann/cardigann#installation :)

Am I missing something?

**Comment by alberto467** [comment]
Yes, i have tried installing it as a service, but when i do so the api key changes and i can't load my custom definitions from `%appdata%/cardigann/definitions`. It seems to have a completely different config folder, that i couldn't find, so i can't run it as a service for now. I tried searching the docs but i didn't find another folder location just for the service, and i searched all of the possible folder locations.

**Comment by aroundmyroom** [comment]
this is in my /etc/system/systemd/Cardigann.service

root@media:/etc/systemd/system# more Cardigann.service
[Unit]
Description=Cardigann Torrent Indexer Proxy
ConditionFileIsExecutable=/usr/local/bin/cardigann

[Service]
StartLimitInterval=5
StartLimitBurst=10
User=root
ExecStart=/usr/local/bin/cardigann "service" "run"





Restart=always
RestartSec=120

[Install]
WantedBy=multi-user.target

**Comment by aroundmyroom** [comment]
working fine !

**Comment by alberto467** [comment]
Unfortunately i run Windows, i think this issue of having two separate config folders for Cardigann as a service and normally is Windows only.

**Comment by tivac** [comment]
Having this issue as well. I've even tried putting `config.json` and my custom definitions in the same folder as `cardigann.exe` without any luck.

**Comment by tivac** [comment]
For kicks I also ensured that `LOCAL SERVICE` (the account that the Cardigann service runs under) has full read access to the cardigann directory. If I run `cardigann.exe server` directly it picks up the config/definition without any issue. Service seems completely unable to find the config though.
